Project.toml

```
@@ -45,7 +45,7 @@ DataFrameMacros = "0.4"
 DataFrames = "1.6"
 Effects = "1"
 KernelDensity = "0.6"
-Loess = "0.5.4" # Loess 0.6 is far too slow
+Loess = "0.5.4"
```

```suggestion
Loess = "0.5.4, 0.6.2"
```

as of 0.6.2, performance is acceptable!

---

I am ok with proposed changes.